src/GafferScene/Preview/RendererAlgo.cpp

```diff
@@ -206,8 +204,7 @@ struct RenderSets::Updater
 
 	void operator()( const tbb::blocked_range<size_t> &r )
 	{
-		ContextPtr context = new Context( *m_context, Context::Borrowed );
-		Context::Scope scopedContext( context.get() );
+		Context::EditableScope context( m_context );
```

Comment:
Cool, I'll update to use SetScope once I've added the update calls. I don't think we're actually missing the removes in this case though, because we're at the outer level of evaluation, so none of the things we'd remove should be in the context anyway.